airlock/business_logic.py

```diff
@@ -611,6 +611,15 @@ def get_outstanding_requests_for_review(self, user: User):
         ],
     }
 
+    STATUS_AUDIT_EVENT = {
+        RequestStatus.PENDING: AuditEventType.REQUEST_CREATE,
```

Comment:
Should we change the status to `.CREATED` for consistency?

Reply:
Maybe -  CREATED doesn't tell you much as user though?